src/test/java/com/google/firebase/messaging/MessageTest.java

```diff
@@ -796,6 +797,99 @@ public void testImageInApnsNotification() throws IOException {
             .build();
     assertJsonEquals(expected, message);
   }
+  
+  @Test
+  public void testInvalidColorInAndroidNotificationLightSettings() throws IOException {
```

Comment:
I think you just need the following bit to trigger the error:

```
LightSettings.builder()
                      .setColorFromString("#01020K")
                      .setLightOnDurationInMillis(1002L)
                      .setLightOffDurationInMillis(1003L)
                      .build();
```

Remove the rest of the boilerplate.

Reply:
Done.